crawl-ref/source/dgnevent.h

```diff
@@ -13,31 +13,31 @@
 // Keep event names in l_dgnevt.cc in sync.
 enum dgn_event_type
 {
-    DET_NONE            = 0x0000,
```

Comment:
Why did you realign (misalign) all these flags without making any meaningful changes?